ow_power_system/src/PowerCSV.cpp

```diff
@@ -0,0 +1,57 @@
+//PowerCSV.cpp
+
+/*The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+Research and Simulation can be found in README.md in the root directory of
+this repository.*/
+
+#include "PowerCSV.h"
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+
+/* Function to load pre-generated values as a vector of vectors of doubles
+ * Returns std::vector<std::vector<std::double>>
+ */
+std::vector<std::vector<double>> powerCSV::loadCSV(const std::string& filename){
+  ifstream power_csv(filename);
+  if (power_csv.fail()) {
+    cout << "Unable to open data file" << endl;
+  }
```

Comment:
Shouldn't the function return here?

Reply:
Yes. Added solution s.t. it should just publish zeros for all values in that case.